cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -253,33 +252,28 @@ func createClusterRoleBindings(clientset *clientset.Clientset) error {
 
 func deletePermissiveNodesBindingWhenUsingNodeAuthorization(clientset *clientset.Clientset, k8sVersion *version.Version) error {
 
-	// If the server version is higher than the Node Authorizer's minimum, try to delete the Group=system:nodes->ClusterRole=system:node binding
-	// which is much more permissive than the Node Authorizer
-	if kubeadmutil.IsNodeAuthorizerSupported(k8sVersion) {
-
-		nodesRoleBinding, err := clientset.RbacV1beta1().ClusterRoleBindings().Get(kubeadmconstants.NodesClusterRoleBinding, metav1.GetOptions{})
-		if err != nil {
-			if apierrors.IsNotFound(err) {
-				// Nothing to do; the RoleBinding doesn't exist
-				return nil
-			}
-			return err
+	nodesRoleBinding, err := clientset.RbacV1beta1().ClusterRoleBindings().Get(kubeadmconstants.NodesClusterRoleBinding, metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// Nothing to do; the RoleBinding doesn't exist
+			return nil
 		}
+		return err
+	}
 
-		newSubjects := []rbac.Subject{}
-		for _, subject := range nodesRoleBinding.Subjects {
-			// Skip the subject that binds to the system:nodes group
-			if subject.Name == kubeadmconstants.NodesGroup && subject.Kind == "Group" {
-				continue
-			}
-			newSubjects = append(newSubjects, subject)
+	newSubjects := []rbac.Subject{}
+	for _, subject := range nodesRoleBinding.Subjects {
+		// Skip the subject that binds to the system:nodes group
+		if subject.Name == kubeadmconstants.NodesGroup && subject.Kind == "Group" {
+			continue
 		}
+		newSubjects = append(newSubjects, subject)
+	}
 
-		nodesRoleBinding.Subjects = newSubjects
+	nodesRoleBinding.Subjects = newSubjects
 
-		if _, err := clientset.RbacV1beta1().ClusterRoleBindings().Update(nodesRoleBinding); err != nil {
-			return err
-		}
+	if _, err := clientset.RbacV1beta1().ClusterRoleBindings().Update(nodesRoleBinding); err != nil {
```

Comment:
In another PR could we add some cookie labels to at least say kubeadm was here.  It always strikes me as odd to proactively change state.